test/test_helper.rb

```diff
@@ -85,6 +87,13 @@ def reconsider_gc_deferment
       end
     end
 
+    def skip_if_plugin_asked_to
+      skips = Foreman::Plugin.tests_to_skip[self.class.name].to_a
+      if skips.any?{|name| @NAME.end_with?(name)}
+        skip "Here be dragons"
```

Comment:
it's not inappropriate, just silly :stuck_out_tongue_closed_eyes: 

anyway... this message isn't actually displayed anywhere, so it's harmless. Every codebase I work with has a "here be dragons" somewhere in it, and it's about time foreman gets one...